pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Attach Amazon EKS Cluster to Kommander
+title: Attach Amazon EKS Cluster to Kommander
+menuWeight: 7
+excerpt: Attach an existing EKS cluster to Kommander
+beta: true 
+enterprise: false
+---
+
+You can attach any existing Kubernetes clusters to Kommander, then use it to examine and manage those clusters later. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+
+## Before you begin 
+
+This procedure requires the following items and configurations:
+
+- A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster.
+- Konvoy v1.5.0-beta.0 or above, [installed and configured](https://docs.d2iq.com/ksphere/konvoy/1.5.0-beta/install/) for your Amazon EKS cluster, on your machine.
+- Kommander v1.0.1-beta.1 or above, [installed and configured](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/install/) on your machine.
```

Comment:
This is a feature for 1.1, but it's in the 1.0 docs??

Reply:
Yes I put it in the wrong folder. I can just dupe this to the 1.1.0-beta equivalent folder, right? Or is it a separate brach like Konvoy is?